audiotool/functions/src/audioapp.ts

```diff
@@ -505,7 +571,7 @@ export class AudioApi {
   async runAdminApiRemoveTasks() {
     const info = this.getBodyJSON();
     const euid = requireParam(info.euid as string);
-    const idTuples = requireArray(info.idTuples as Array<[string, string]>, 1);  // list of [taskSetId, taskId]
+    const idTuples = requireArray(info.idTuples as Array<[string, string]>, 1);  // list of [taskSetId, user taskId]
```

Comment:
The comment change led me to expect Array<[string, string, string]>.

Reply:
Rephrased, thank you for the second pair of eyes